docs/user/choosing-a-site.rst

```diff
@@ -1,41 +1,47 @@
-Choosing Between Our Two Sites
-==============================
+Choosing Between Our Two Platforms
+==================================
 
-A question our users often have is what the difference is between |org_brand| and |com_brand|.
-This page will lay out the functional and philosophical differences between the two sites,
-which should help you choose which is a better fit for your organization.
+Users often ask what the differences are between |org_brand| and |com_brand|.
 
-The features available on both platforms are the same.
-The primary difference is the audience and use cases that are supported.
+While many of our features are available on both of these platforms, there
+are some key differences between our two platforms. 
 
 Read the Docs Community
 -----------------------
 
-|org_brand| is meant for open source projects to use for documentation hosting.
-This is great for user and developer documentation for your project.
+|org_brand| is strictly for hosting open source documentation. This means that
+project documentation source, and the generated documentation, are both publicly
+accessible and licensed under a permissive license.
```

Comment:
Why permissive specifically? Could be copyleft as well (not considered "permissive" as far as I understand). Or, not have a license at all, because people tend to forget to choose a license. I would remove references to the licensing here, just say that the sources are "publicly visible" or something like it.

Reply:
Same, I don't think we mention any of that in our ToS

Reply:
Good points.

I wasn't meaning to differentiate between copyleft and permissive here. There's no point where we check or enforce licensing, however we do describe in several places (or have described in the past) that community is for open source projects or non-commercial works. Dropping mention of a license is the most correct though.

True about repositories without a license. Github TOS would apply here by default, meaning those repositories retain full copyright and do not explicitly grant us any rights at all -- even the right "to use".

In this case, our TOS does grant us necessary rights:
https://docs.readthedocs.io/en/stable/terms-of-service.html#license-grant-to-us

So, open licensing is not a legal requirement because our TOS works around that. But, I will leave the mention of open source documentation hosting.